### PR TITLE
Updated readme and community health files

### PR DESCRIPTION
**Pull request by Beak-man**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -160,5 +159,5 @@ For Web WorldWind tutorials and examples, please check out our website: https://
 
 For community support and FAQs, check out the WorldWind Forum: https://forum.worldwindcentral.com/.
 
-To reach our Administrative Contact, please email: [patrick.hogan@nasa.gov](mailto:patrick.hogan@nasa.gov).
+To reach our Administrative Contact, please email: [patrick.hogan@nasa.gov](arc-worldwind@mail.nasa.gov).
```
Comment (markpet49) [inline CONTRIBUTING.md:162]: Remove Patrick's name.

File: README.md [hunk 2]
```diff
@@ -4,29 +4,44 @@
...
+- Updated development dependencies to latest versions
+
+WorldWind's API remains largely unchanged in this release and we are committed to maintaining a consistent API in future releases.
+
+More information on the release can be found at these links: [Web WorldWind 0.11.0](https://github.com/NASAWorldWind/WebWorldWind/releases).
 Please direct questions to our new email address: 
 
 arc-worldwind@mail.nasa.gov
 
 [![Build Status](https://travis-ci.com/NASAWorldWind/WebWorldWind.svg?branch=develop)](https://travis-ci.com/NASAWorldWind/WebWorldWind)
```
Comment (markpet49) [inline README.md:27]: This link doesn't seem to work.

File: README.md [hunk 3]
```diff
@@ -4,29 +4,44 @@
 
 ## New versions of WorldWind released
  
-Web WorldWind 0.10.0 and WorldWind Java 2.2.0 are now available on GitHub. The new version of Web WorldWind addresses potential vulnerabilities in the code and underlying packages. The new version of WorldWind Java focuses on upgrading to Java 11 and JOGL 2.4 as well as a switch to Apache 2.0 license. WorldWind's API remains largely unchanged in this release and we are committed to maintaining a consistent API in future releases.
-More information on the release can be found at these links: [Web WorldWind 0.10.0](https://github.com/NASAWorldWind/WebWorldWind/releases) and [WorldWind Java 2.2.0](https://github.com/NASAWorldWind/WorldWindJava/releases).
+Web WorldWind 0.11.0 is now available on GitHub. The summary of changes is as follows:
+- Improvements to COLLADA 3D model support.
```
Comment (markpet49) [inline README.md:8]: These bullet points didn't work for me in a markdown viewer. Adding a carriage return before the first bullet fixed it for me.

**Review by markpet49 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Beak-man) [reply README.md:8]: It's fixed now.

File: README.md (hunk 2 above)
Comment (Beak-man) [reply README.md]: I changed the phrasing to convey that the link leads to the release list, which was the original intent.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Beak-man) [reply CONTRIBUTING.md]: It's fixed now. I added 'mailto:' to the links.

**Review by markpet49 — APPROVED** [review]
Looks good.

**Pull request merged**